src/ui/transforms.js

```diff
@@ -58,22 +58,22 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
     });
 
     sjrk.storyTelling.transforms.arrayToString = function (input, extraInputs) {
-        var contentString = "";
-        var separator = extraInputs.separator();
-        var stringOnly = extraInputs.stringOnly();
-        var path = extraInputs.path();
+        var combinedString = "";
+        var separator = extraInputs.separator(),
+            stringOnly = extraInputs.stringOnly(),
+            path = extraInputs.path();
 
         fluid.each(input, function (term) {
             if (path) {
                 term = fluid.get(term, path) || term;
             }
```

Comment:
It looks like `path` defaults to "", can you right this whole block as `term = fluid.get(term, path)`? Also it looks like `stringOnly` defaults to `""` but should probably default to `false`.